snooty/test_semantic_parser.py

```diff
@@ -94,18 +97,40 @@ def test_expand_includes(backend: Backend) -> None:
 def test_validate_ref_targets(backend: Backend) -> None:
     page_id = FileId("refrole.txt")
     ast = cast(Dict[str, List[SerializableType]], backend.pages[page_id].ast)
+
+    # Assert that targets found in intersphinx inventories are correctly resolved
     paragraph = cast(Dict[str, Any], ast["children"][0])
     ref_role = paragraph["children"][0]
-    assert ref_role["type"] == "ref_role"
-    assert "url" in ref_role
-    assert (
-        ref_role["url"]
-        == "https://docs.mongodb.com/manual/reference/configuration-options/#net.port"
+    print(ast_to_testing_string(ref_role))
+    check_ast_testing_string(
```

Comment:
Thanks for this improvement 🙌 I think there are other places where it could be used in this file as well. Alas!